Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -6,21 +6,17 @@
 #include <cstdio>
 #include <cstdlib>
 #include <ctime>
-#include <hidsdi.h>
 #include <unordered_map>
 #include <unordered_set>
 
-// clang-format off
 #include <windows.h>
-// The following Windows headers must be included AFTER windows.h.
+#include <initguid.h>
```

Comment:
Maybe you could just move the Devpkey.h in another block down to prevent sorting with the rest. All the other headers beside Windows.h should be possible to sort normally. Then you wouldn't need to single out initguid.h in the clang format configuration either (the Windows.h exception is good!)